src/plugins/rel/rel.js

```diff
@@ -159,14 +164,26 @@
                 el: el,
                 x: el.getAttribute( "data-x" ),
                 y: el.getAttribute( "data-y" ),
-                z: el.getAttribute( "data-z" )
+                z: el.getAttribute( "data-z" ),
+                rotateX: el.getAttribute( "data-rotate-x" ),
+                rotateY: el.getAttribute( "data-rotate-y" ),
+                rotateZ: el.getAttribute( "data-rotate-z" )
             } );
             var step = computeRelativePositions( el, prev );
 
             // Apply relative position (if non-zero)
             el.setAttribute( "data-x", step.x );
             el.setAttribute( "data-y", step.y );
             el.setAttribute( "data-z", step.z );
+            el.setAttribute( "data-rotate-x", step.rotateX );
+            el.setAttribute( "data-rotate-y", step.rotateY );
+            el.setAttribute( "data-rotate-z", step.rotateZ );
+            el.removeAttribute( "data-rel-x" );
+            el.removeAttribute( "data-rel-y" );
+            el.removeAttribute( "data-rel-z" );
+            el.removeAttribute( "data-rel-rotate-x" );
+            el.removeAttribute( "data-rel-rotate-y" );
+            el.removeAttribute( "data-rel-rotate-z" );
```

Comment:
Why are the removeAttribute()s needed? Note that if you want to remove some attribute, you must reset the original value in the lib.gc.pushCallback() function.